docs/resources/privileged_access_group_eligibility_schedule_request.md

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Identity Governance"
+---
+
+# Resource: azuread_privileged_access_group_eligibility_schedule_request
```

Comment:
Ref https://github.com/hashicorp/terraform-provider-azuread/pull/1327/files#r1552686123

Reply:
I've moved to the `azuread_privileged_access_group_eligibility_schedule` resource are recommended. I've catered (I think) for both future start dates, and expiry of a time-bound schedule